Source/Plugins/Tilemaps/Core/Tilesets/Tileset.cs

```diff
@@ -298,8 +298,8 @@ public int GetCompileHashCode()
 				MathF.CombineHashCode(ref hash, input.Id.GetHashCode());
 				MathF.CombineHashCode(ref hash, input.Name.GetHashCode());
 				MathF.CombineHashCode(ref hash, input.SourceData.GetHashCode());
-				MathF.CombineHashCode(ref hash, input.SourceTileSize.GetHashCode());
-				MathF.CombineHashCode(ref hash, input.SourceTileSpacing.GetHashCode());
+				MathF.CombineHashCode(ref hash, input.SourceTileSize.GetHashCode() + input.SourceTileSize.X * input.SourceTileSize.Y);
+				MathF.CombineHashCode(ref hash, input.SourceTileSpacing.GetHashCode() + input.SourceTileSize.X * input.SourceTileSize.Y);
```

Comment:
There seems to be a copy-paste bug, as the spacing hash calculation uses the tile size struct.